src/test/resources/com/puppycrawl/tools/checkstyle/checks/header/regexpheader/InputRegexpSmallHeader.java

```diff
@@ -0,0 +1,6 @@
+package com.puppycrawl.tools.checkstyle.checks.header.regexpheader;
+
+
+/**
+ */
+public class InputRegexpSmallHeader {}
```

Comment:
Inputs must be named after their tests. Test is `RegexpHeaderCheckTest` so inputs' name must start with `InputRegexpHeader`.
Please also keep class name as file name.

Please also do the same with the `regexp.header` files.